### ISSUE TITLE
TLS accept client certificate failed 

### ISSUE THREAD

**Issue by Lexus89** [sensepost/hostapd-mana#24]
I have been testing the functionality lately and it definately has improved alot! 

When reading the wiki i saw a function that was not mentioned in the default hostapd.conf provided with the mana functionality (mana_eaptls=1). Creating the following wifi profile on my android device  (8.0.0)  still results in failed authentication.

- Using TLS
- Do NOT validate server certificate
- Do NOT provide a client certificate

**Comment by singe** [comment]
Heya. It’s got a little write up at https://github.com/sensepost/hostapd-mana/wiki/MANA-EAP-Options-(aka-WPE)#eap-tls-mitm and https://github.com/sensepost/hostapd-mana/wiki/EAP-WPE-Attack-Theory#eap-tls as of yesterday ;)

I’m confused though, you said you configured your Android to not send a client cert? EAP-TLS requires a client cert?

**Comment by Lexus89** [comment]
Yeah weird enough my android wifi profile setup let's me connect without providing a client certificate..

**Comment by Lexus89** [comment]
Thanks singe! I'm going to play with this abit more.
